### PR TITLE
Addition of some information on using systemd to start scripts at boot

### PR DESCRIPTION
**Pull request by buxtonpaul**
Based on some discussion on the forums I have created some guidance on using systemd to start scripts as once configured properly is a better way of handling services. My first attempt at contributing to things :-)

### CODE REVIEW

**Comment by JamesH65** [comment]
Sorry about delay, need to tech check this when I get the time.

File: linux/usage/systemd.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+On your Pi, create a .service file for your service e.g.
+
+myscript.service
+
+```[Unit]
+Description=My service
+After=network.target
+
+[Service]
+ExecStart=/usr/bin/python3 -u main.py
```
Comment (JamesH65) [inline linux/usage/systemd.md]: One thing, you probably need to explain that service you are starting is, in this example, a Python program, but that you could have any sort of application start up here.

Comment (buxtonpaul) [inline linux/usage/systemd.md]: Agree, that makes sense.

**Comment by JamesH65** [comment]
Tech check done, sending to copy edit

**Pull request merged**